core/jobs/batch_jobs/firebase_sync_jobs.py

```
@@ -0,0 +1,142 @@
+# coding: utf-8
```

Ruff (0.15.15)

[warning] UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)